tests/test_ext4.py

```diff
@@ -80,25 +77,11 @@ def test_sparse(ext4_sparse_bin: BinaryIO) -> None:
     assert sparse_all.dataruns() == [(None, 5120)]
 
 
-@pytest.mark.parametrize(
-    "image_file",
-    [
-        ("tests/data/ext4_symlink_test1.bin.gz"),
-        ("tests/data/ext4_symlink_test2.bin.gz"),
-        ("tests/data/ext4_symlink_test3.bin.gz"),
-    ],
-)
-def test_symlinks(image_file: str) -> None:
+def test_symlinks(ext4_symlink_bin: BinaryIO) -> None:
     path = "/path/to/dir/with/file.ext"
-    expect = b"resolved!\n"
-
-    def resolve(node: INode) -> INode:
-        while node.filetype == stat.S_IFLNK:
-            node = node.link_inode
-        return node
 
-    with gzip.open(image_file, "rb") as disk:
-        assert resolve(ExtFS(disk).get(path)).open().read() == expect
+    extfs = ExtFS(ext4_symlink_bin)
```

Comment:
Reviewer: re-purposed test to verify link property now that "local" symlink lookup is removed.